datafusion/core/tests/sqllogictests/src/main.rs

```diff
@@ -47,13 +47,7 @@ pub async fn main() -> Result<(), Box<dyn Error>> {
 
     let options = Options::new();
 
-    let files: Vec<_> = read_test_files(&options);
-
-    info!("Running test files {:?}", files);
-
-    for path in files {
-        let file_name = path.file_name().unwrap().to_str().unwrap().to_string();
-
+    for (path, file_name) in read_test_files(&options) {
```

Comment:
`file_name` is technically not a file name here, but `relative_file_path`/`relative_file_path`/`relative_file_name`. 
I haven't changed the name yet, but it could be clearer to explain somehow that the string is relative to the root directory.